### ISSUE TITLE
VB -> C#: Localized designer files not converted

### ISSUE THREAD

**Issue by mrmonday** [icsharpcode/CodeConverter#423]
### Input code

Create a new .NET Core 3.0 library (also applies to .NET Framework, with the relevant project file). Add two resource files, `Resource1.resx` and `Resource1.en-US.resx`.

Use the following project file:

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Library</OutputType>
    <RootNamespace></RootNamespace>
    <TargetFramework>netcoreapp3.0</TargetFramework>
    <DefaultItemExcludes>$(DefaultItemExcludes);$(ProjectDir)**\*.resx;$(ProjectDir)**\*.Designer.vb</DefaultItemExcludes>
  </PropertyGroup>

  <ItemGroup>
    <Compile Include="Resource1.Designer.vb">
      <DesignTime>True</DesignTime>
      <AutoGen>True</AutoGen>
      <DependentUpon>Resource1.resx</DependentUpon>
    </Compile>
  </ItemGroup>
  <ItemGroup>
    <EmbeddedResource Include="Resource1.resx">
      <CustomToolNamespace>My.Resources</CustomToolNamespace>
      <Generator>ResXFileCodeGenerator</Generator>
      <LastGenOutput>Resource1.Designer.vb</LastGenOutput>
    </EmbeddedResource>
  </ItemGroup>

  <ItemGroup>
    <Compile Include="Resource1.en-US.Designer.vb">
      <DesignTime>True</DesignTime>
      <AutoGen>True</AutoGen>
      <DependentUpon>Resource1.en-US.resx</DependentUpon>
    </Compile>
  </ItemGroup>
  <ItemGroup>
    <EmbeddedResource Include="Resource1.en-US.resx">
      <CustomToolNamespace>My.Resources</CustomToolNamespace>
      <Generator>ResXFileCodeGenerator</Generator>
      <LastGenOutput>Resource1.en-US.Designer.vb</LastGenOutput>
    </EmbeddedResource>
  </ItemGroup>

</Project>
```

### Erroneous output

The file `Resource1.en-US.Designer.cs` is not created, despite the corresponding VB file existing. Manually creating the file causes the contents to be generated correctly.

### Expected output

The designer file should exist. The non-localised one does.

### Details

* Version in use: 7.2.0

**Comment by mrmonday** [comment]
There's a related issue to this with the namespaces being different, I haven't managed to work out a test case though.

**Comment by GrahamTheCoder** [comment]
I believe I've fixed the namespace issue as part of #545
I'll look into the missing file issue now

**Comment by Saibamen** [comment]
Should be fixed in release 8.0.2: https://github.com/icsharpcode/CodeConverter/releases/tag/8.0.2
Please retest
